### PR TITLE
Extract prod

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR, for backend:
 - enhances Identity view in admin
 - changes default extract user to sitn_extract
 - reverts WKTLatLong to WKT (Long, Lat), don't ask...
 - handles better order_items when a PUT or PATCH is done on /orders

In front:
 - larger cart overlay
 - `DELETE` on orderitem when you press the delete button in cart
 - replaces calls to `/product/?search=<product_label>` by `/product/<product_id>`
 - deletes unused old identity properties
 - do not let user add the same product twice in cart
 - adds possibility to change formats for all items in cart:
 
image

### CODE REVIEW

**Review by kalbermattenm — APPROVED** [review]
Pfiou, that's quite a PR... I really enjoyed the WKT long/lat <-> lat/long war. I just hope it is definitively over ;-)

Maybe just one thing for the "allavailableformats". If two products do not have matching formats, then it is not possible to set the same format everywhere, right? I somehow missed that point while reviewing and I was wondering how it was implemented...

File: front/src/app/account/new-order/new-order.component.ts [hunk 1]
```diff
@@ -251,24 +256,33 @@ export class NewOrderComponent implements OnInit, OnDestroy {
...
+        if (this.orderItemFormGroup.controls[attr]) {
+          this.orderItemFormGroup.removeControl(attr);
+        }
       }
     }
 
-    order.items.forEach((item) => {
+    this.allAvailableFormats = new Set();
+    for (const item of order.items) {
+      item.available_formats?.forEach(format => this.allAvailableFormats.add(format));
```
Comment (maltaesousa) [inline front/src/app/account/new-order/new-order.component.ts:269]: @kalbermattenm , here, for each order_item, add all available_formats to the allAvailableFormats Set

File: front/src/app/account/new-order/new-order.component.ts [hunk 2]
```diff
@@ -450,6 +464,22 @@ export class NewOrderComponent implements OnInit, OnDestroy {
     return `${orderItem.id}_${Order.getProductLabel(orderItem)}`;
   }
 
+  updateAllDataFormats() {
+    this.isOrderPatchLoading = true;
+    const dataFormatName = this.orderItemFormGroup.get('formatsForAll')?.value || '';
+    for (const item of this.currentOrder.items) {
+      const availableFormats = item.available_formats || [];
+      if (availableFormats.indexOf(dataFormatName) > -1) {
```
Comment (maltaesousa) [inline front/src/app/account/new-order/new-order.component.ts:472]: @kalbermattenm 
Here, when customer chooses a format, if the format is not available for the current item, nothing happens. The customer will have to specify it manually if he wants to submit order.

**Comment by kalbermattenm** [comment]
Thanks for the explanation!

**Pull request merged**